rules/windows/privilege_escalation_windows_service_via_unusual_client.toml

```diff
@@ -44,14 +44,17 @@ Audit Security System Extension (Success)
 severity = "high"
 tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Privilege Escalation"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.action:"service-installed" and
-  (winlog.event_data.ClientProcessId:"0" or winlog.event_data.ParentProcessId:"0") and
+configuration where host.os.type == "windows" and
+  event.action == "service-installed" and
+  (winlog.event_data.ClientProcessId == "0" or winlog.event_data.ParentProcessId == "0") and
   not winlog.event_data.ServiceFileName : (
-    "C:\\Windows\\VeeamVssSupport\\VeeamGuestHelper.exe" or
-    "%SystemRoot%\\system32\\Drivers\\Crowdstrike\\17706-CsInstallerService.exe"
+    "?:\\Windows\\VeeamVssSupport\\VeeamGuestHelper.exe",
+    "?:\\Windows\\VeeamLogShipper\\VeeamLogShipper.exe",
+    "%SystemRoot%\\system32\\Drivers\\Crowdstrike\\*-CsInstallerService.exe",
+    "\"%windir%\\AdminArsenal\\PDQInventory-Scanner\\service-1\\PDQInventory-Scanner-1.exe\" "
```

Comment:
Interesting exclusion right there, especially with the space in the end. I assume it's intended?

Reply:
Yup